docs/requirements/infrastructure-requirements.md

```diff
@@ -0,0 +1,274 @@
+# CLARIAH Requirements for Application Infrastructure
+
+* [Introduction](#introduction)
+* [Principles](#principles)
+* [Definitions](#definitions)
+* [Requirements](#requirements)
+
+## Introduction
+
+This set of requirements is largely derived from [earlier work](https://github.com/netwerk-digitaal-erfgoed/requirements-infrastructure/) done in the scope of the Netwerk Digitaal Erfgoed (NDE).
+
+CLARIAH aims to develop a service-oriented infrastructure consisting of a wide variety of different applications, this
+is often denotes as CLARIAH-as-a-Service (CLaaS) for short.
+
+The user adoption of applications depends not only on their usefulness and quality,
+but also on their reliability and performance.
+When applications are slow or buggy, users will drop out.
+For foundational services such as those delivered by CLARIAH, on which large groups of users depend,
+this problem is even more prominent.
+
+Infrastructure plays an important role in delivering reliable software.
+This document describes requirements for a fault tolerant, performant and reliable infrastructure.
+The requirements are based on modern DevOps and other best practices, including [Twelve-Factor](https://12factor.net)
+and [FAIR4RS](https://www.rd-alliance.org/system/files/FAIR4RS_Principles_v0.3_RDA-RFC.pdf).
+The [current NDE infrastructure](https://github.com/netwerk-digitaal-erfgoed/infrastructure) is based on these requirements and CLARIAH would do good to follow suit here.
+
+This document is aimed at organisations and the IT operators within it that would like to take ownership of (‘borgen’) CLARIAH applications.
```

Comment:
```suggestion
This document is aimed at organisations and their IT operators that would like to take ownership of (‘borgen’) CLARIAH applications.
```

The term ‘borgen’ may be too NDE-specific, so I suggest removing it here. Perhaps: ‘that would like to provide CLaaS components’?